airflow/providers/openlineage/provider.yaml

```diff
@@ -59,6 +59,13 @@ config:
         example: ~
         default: "False"
         version_added: ~
+      disabled_for_operators:
+        description: |
+          Comma separated string of Airflow Operator names to disable
+        type: boolean
+        example: "BashOperator,PythonOperatorOperator"
+        default: ""
+        version_added: ~
```

Comment:
Here we should add open-lineage version in which it will be added. Which I think should be 1.1.0 as this is a new feature.